app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::ReviewsController < ApiController
+  def new
+    new_review = Review.new
+  end
+
+  def create
+    new_review = Review.new(review_params)
+    new_review.user = current_user
+
+    if new_review.save
+      render json: { review: new_review }
+    else
+      render json: {errors: new_review.errors }
```

Comment:
I'd recommend including a 422 status code here for "unprocessible entity" - it can give the client a clue on whether the `create` was successful (200) or not (422)